[Bug] POST /api/objects/ error when adding relatives via pedigree view

### Bug Description

Description:
Hi, I am running the latest Gramps Web Docker image with Python 3.13. I've encountered two issues that seem related to the recent environment and API changes.

1. Startup Issue:
The container fails to start unless GRAMPSWEB_TREE is explicitly defined in the environment variables, throwing a ValueError: TREE must be specified. While this might be intended, it's a change from previous versions where the initial setup could be completed without it.

2. API Error (POST /api/objects/):
When I am in the pedigree/tree view and try to add a relative (e.g., adding a spouse to a sibling), the request fails.

    Observation: Adding a person via the "People" list menu works fine.

    Problem: Adding via the "Quick Add" button in the tree view results in a 500 Internal Server Error.

Traceback from logs:
Plaintext

[ERROR] Error handling request POST /api/objects/
Traceback (most recent call last):
  File "/venv/lib/python3.13/site-packages/gramps_webapi/api/resources/objects.py", line 83, in post
    objects = self._parse_objects()
  File "/venv/lib/python3.13/site-packages/gramps_webapi/api/resources/objects.py", line 70, in _parse_objects
    obj_dict = fix_object_dict(obj_dict)
  File "/venv/lib/python3.13/site-packages/gramps_webapi/api/resources/util.py", line 1319, in fix_object_dict
    obj = gramps.gen.lib.__dict__[_class]()
KeyError: (Missing or incorrect _class key)

Environment:

    Deployment: Docker (LXC on Proxmox)

    Version: ghcr.io/gramps-project/grampsweb:latest

    Python Version: 3.13

    Browser: Reproduced in Firefox (Cache cleared), works partially in Chrome for simple additions, but fails for complex relational additions in the tree view.

It seems like the frontend might be sending an object format that the new Python 3.13 backend API does not correctly map to the internal Gramps classes anymore.

Gramps 6.0.8
Gramps Web API 3.13.0
Gramps Web Frontend 26.5.1
Gramps QL 0.4.0
Sifts 1.3.1
locale: en
multi-tree: false
task queue: false
OCR: true
chat: false

Thank you!

### Gramps Web Version Information

Gramps 6.0.8
Gramps Web API 3.13.0
Gramps Web Frontend 26.5.1
Gramps QL 0.4.0
Sifts 1.3.1
locale: en
multi-tree: false
task queue: false
OCR: true
chat: false

### Verification

- [x] I have searched for existing issues before creating this one

--- a/src/mixins/GrampsjsNewPersonMixin.js
+++ b/src/mixins/GrampsjsNewPersonMixin.js
@@ -166,7 +166,14 @@ export const GrampsjsNewPersonMixin = superClass =>
       const deathString = this.translateTypeName(false, 'event_types', 'Death')
       // type is the family relation type set by GrampsjsFormNewSpouse; strip it
       // so it doesn't leak into the Person payload (_type silences unused-var lint)
-      const {birth = {}, death = {}, frel, mrel, type: _type, ...person} = this.data
+      const {
+        birth = {},
+        death = {},
+        frel,
+        mrel,
+        type: _type,
+        ...person
+      } = this.data
       const hasBirth = birth.place || (birth?.date && !dateIsEmpty(birth.date))
       const hasDeath = death.place || (death?.date && !dateIsEmpty(death.date))
       if (!hasBirth && !hasDeath) {

--- a/src/mixins/GrampsjsNewPersonMixin.js
+++ b/src/mixins/GrampsjsNewPersonMixin.js
@@ -164,7 +164,9 @@ export const GrampsjsNewPersonMixin = superClass =>
       const handleDeath = makeHandle()
       const birthString = this.translateTypeName(false, 'event_types', 'Birth')
       const deathString = this.translateTypeName(false, 'event_types', 'Death')
-      const {birth, death, frel, mrel, ...person} = this.data
+      // type is the family relation type set by GrampsjsFormNewSpouse; strip it
+      // so it doesn't leak into the Person payload (_type silences unused-var lint)
+      const {birth, death, frel, mrel, type: _type, ...person} = this.data
       const hasBirth = birth.place || (birth?.date && !dateIsEmpty(birth.date))
       const hasDeath = death.place || (death?.date && !dateIsEmpty(death.date))
       if (!hasBirth && !hasDeath) {

--- a/src/mixins/GrampsjsNewPersonMixin.js
+++ b/src/mixins/GrampsjsNewPersonMixin.js
@@ -166,7 +166,7 @@ export const GrampsjsNewPersonMixin = superClass =>
       const deathString = this.translateTypeName(false, 'event_types', 'Death')
       // type is the family relation type set by GrampsjsFormNewSpouse; strip it
       // so it doesn't leak into the Person payload (_type silences unused-var lint)
-      const {birth, death, frel, mrel, type: _type, ...person} = this.data
+      const {birth = {}, death = {}, frel, mrel, type: _type, ...person} = this.data
       const hasBirth = birth.place || (birth?.date && !dateIsEmpty(birth.date))
       const hasDeath = death.place || (death?.date && !dateIsEmpty(death.date))
       if (!hasBirth && !hasDeath) {

